docs/documentation/docs/controls/Dashboard.md

```diff
@@ -142,6 +148,9 @@ Provides Widget link properties
 | Property | Type | Required | Description |
 | ---- | ---- | ---- | ---- |
 | href | string | yes | Link to be opened. |
+| title | string | no | The text to display for the link, if not provided, the default text will be used. |
+| color | string | no | The color of the link, if not provided, the "default" color will be used. The available colors can be found on the [official Fluent UI documentation of the Text control](https://fluentsite.z22.web.core.windows.net/0.66.2/components/text/definition#variations-color). |
```

Comment:
Please update the property description as discussed [here](https://github.com/pnp/sp-dev-fx-controls-react/pull/1813#discussion_r1712485469) (`brand` instead of `default`).

Reply:
The code is already updated so that the default value is `default` instead of `brand`.

Reply:
Indeed, my mistake!